src/scry/mtg.ts

```diff
@@ -193,9 +193,10 @@ export class Deck {
       }
     }
 
-    if (!this.checkCardsSum(60, this.main, ">=")) {
+    const sum = this.main.reduce<number>(this.sumer, 0);
+    if (sum != 40 && sum < 60) {
```

Comment:
This allow a player to submit a 40 card constructed deck. Is there a way to specify deck format, so better check can be done?
You could add a "deck content check" rule, that could verify if the deck is correct based on foramt (singleton, brawl, cosntructed, historic, forgeron, limited, etc)

Reply:
Yes, it's a planned feature !